src/api/DataHandler.ts

```diff
@@ -21,5 +71,5 @@ async function fetchData(url: string, requestType: HttpRequestType, bodyData?: s
 }
 
 export function createNewPost(post: object) {
```

Comment:
Consistancy with naming conventions, capital letter for function name, as the previous.

Reply:
should be resolved with recent commit